spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala

```diff
@@ -340,7 +340,7 @@ class CometSparkSessionExtensions
               op
           }
 
-        case op: LocalLimitExec =>
+        case op: LocalLimitExec if getOffset(op) == 0 =>
```

Comment:
Add a `withInfo` for the case when `offset > 0` ?

Reply:
Oh, I missed it. I will create a follow up. Thanks.